pages/k8s/release-notes.md

```diff
@@ -14,6 +14,89 @@ toc: False
+
+With this release, Charmed Kubernetes can be deployed with a
+juju 2.9, 3.0 or 3.1 controller. This release is intended to serve
+as a migration point away from juju 2.9 deployments which is why we 
+offer a tested strategy of our charms on both 2.9 and 3.1 releases.
+
+## Technical Previews
+
+### KubeVirt
+Offered as a technical preview, the KubeVirt charm offers an opinionated deployment of KubeVirt on Charmed Kubernetes such that virtual machines can be launched within your Kubernetes cluster.  Charmed Kubernetes on metal will deploy KubeVirt in such a way to use faster, native hardware virtualization, but KubeVirt also supports software emulation for cases where accelerated hardware support is not available. 
+
+### Volcano Scheduler
+Offered as a technical preview, the suite of Volcano charms deploys on either MicroK8s or Charmed Kubernetes, and can be used to more effectively schedule ML/AI workloads which need to ensure effective queuing of jobs requiring GPU resources. The charm ships with v1.7.0 of Volcano and will follow future upstream releases.
+
+### Cluster API Providers
+Cluster API providers for deploying Charmed Kubernetes are nearing completion and available as technical previews. These providers consist of the Juju Infrastructure Provider, the CharmedK8s Control Plane Provider, and the CharmedK8s Bootstrap Provider. 
+
+The infrastructure provider is responsible for Juju model management and machine deployment. The control plane provider handles control plane management, kubeconfig management, and control plane status reporting. The bootstrap provider controls what charms are deployed to the machines provisioned by the infrastructure provider. 
+
+While the user experience surrounding certain Juju-related interactions is still being improved, Charmed Kubernetes can be deployed using the familiar Cluster API workflow using the providers in their current state.
+
+
+## Component Versions
+
+### Charm/Addons pinned versions
+- kube-ovn 1.10.4
+- calico 3.21.4
+- cephcsi 3.7.2
+- cinder-csi-plugin 1.26.2
+- coredns 1.9.4
+- ingress-nginx 1.6.4
+- k8s-keystone-auth 1.26.2
+- kube-state-metrics 2.8.2
+- kubernetes-dashboard 2.7.0
+- openstack-cloud-controller-manager 1.26.2
+
+### Charm default versions
+- cloud-provider-vsphere 1.26
+- vsphere-csi-driver 3.0.0
+- cloud-provider-azure 1.25.0
+- azuredisk-csi-driver 1.23.0
+- cloud-provider-aws 1.26.1
+- aws-ebs-csi-driver 1.12.0
+- gcp-compute-persistent-disk-csi-driver 1.8.0
+
+ 
+## Fixes
+
+A list of bug fixes and other minor feature updates in this release can be found at
+[the launchpad milestone page for 1.27](https://launchpad.net/charmed-kubernetes/+milestone/1.27).
+
+
+## Notes and Known Issues
+
+
+## Deprecations and API changes
+
+- Upstream
+
+For details of other deprecation notices and API changes for Kubernetes 1.27, please see the
+relevant sections of the [upstream release notes][upstream-changelog-1.27].
+
+<!-- wokeignore:rule=master -->
```

Comment:
can we lose this from the template - it breaks the webteam's parser if there isn't a blank line after HTML elements (which of course, makes the ignore command not ignore the bit it is supposed to

Reply:
we can either drop it from the templates and drop it from the rendered but it will make the woke checks fail on the repo for just this PR.  Afterwards it should be okay

Reply:
i could put a blanket ignore of the word `master` on this repo... we can just stop checking for it